docs/mining/how-to-stake.md

```diff
@@ -1,37 +1,40 @@
-# <img class="dcr-icon" src="/img/dcr-icons/QuestionTicket.svg" /> How to Stake
+# <img class="dcr-icon" src="/img/dcr-icons/QuestionTicket.svg" /> How to Stake/Vote
+
+Last updated for v1.3.0
```

Comment:
No need to add this here, I think this may be a rebasing issue

Reply:
I assume you mean remove `Last updated for v1.3.0`, which I've done.